README.md

```diff
@@ -142,6 +142,7 @@ You can override the following variables in the plugin configuration:
     * longType
     * nullType
     * stringType
+    * timestampMillisType - possible values are `JavaTimeInstant` or `JavaSqlTimestamp`
```

Comment:
I had a look at https://github.com/julianpeeters/avrohugger/blob/master/avrohugger-core/src/main/scala/types/LogicalAvroScalaTypes.scala and saw that these types are called `LogicalAvroScalaTypes`. Could you add another List with `Logical` next to `Primitive`?

Reply:
addressed.